tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -519,6 +535,105 @@ def test_bounce(self, clean, connect_protocol):
 
         assert success, "Found validation errors:\n" + "\n  ".join(errors)
 
+    @cluster(num_nodes=6)
+    @matrix(clean=[True, False], connect_protocol=['sessioned', 'compatible', 'eager'])
+    def test_exactly_once_source(self, clean, connect_protocol):
+        """
+        Validates that source and sink tasks that run continuously and produce a predictable sequence of messages
```

Comment:
1. Did we validate `sink` tasks in the test?
2. Should we mention "exactly once" here?

Reply:
1. Good point, no, we do not.
2. I believe it is already mentioned? I've cleaned up the description a bit but left in the "deliver messages exactly once" part.